addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -454,11 +454,11 @@ var SnippetEditor = Widget.extend({
      * @param {boolean} [show]
      * @returns {Promise<boolean>}
      */
-    toggleTargetVisibility: async function (show) {
+    toggleTargetVisibility: async function (show, willSave) {
```

Comment:
- Added a parameter but not in the documentation just above
- In stable, I would avoid adding parameters. And if really necessary, I would make it an object so that custo can still add their own object as parameters (kinda like `kwargs` in python). But in the end, I don't think this parameter is necessary.